sockeye/arguments.py

```diff
@@ -645,8 +672,14 @@ def add_model_parameters(params):
                                    '(validation) source factor files. Default: %(default)s.')
     model_params.add_argument('--source-factors-combine', '-sfc',
                               choices=C.SOURCE_FACTORS_COMBINE_CHOICES,
-                              default=C.SOURCE_FACTORS_COMBINE_CONCAT,
+                              default=[C.SOURCE_FACTORS_COMBINE_CONCAT],
+                              nargs='+',
                               help='How to combine source factors. Default: %(default)s.')
+    model_params.add_argument('--source-factors-share-embedding',
+                              type=bool_str(),
+                              nargs='+',
+                              default=[False],
+                              help='Share the embeddings with the source language. Default: do not share.')
```

Comment:
could you add a sentence in help to indicate that this is a list?

Reply:
Done, although the help produced by argparse already shows this.